start.java

```diff
@@ -53,6 +58,7 @@ public static void main(String[] args) {
             System.out.printf("  %s: %s%n", HTTPS_OPT, "enables HTTPS support. (Optional) Implies --http. If not provided, plain HTTP is used");
             System.out.printf("  %s: %s%n", OPENLDAP_OPT, "enables OpenLDAP support. (Optional)");
             System.out.printf("  %s: %s%n", POSTGRES_OPT, "enables postgrase database support. (Optional) If no other database is provided, H2 database is used");
+            System.out.printf("  %s: %s%n", EXTENSIONS_OPT, "enables deployemnt style, choose hot (default) or package depending on your needs");
```

Comment:
"enables deployemnt style, choose hot (default) or package depending on your needs" -> This sound strange: we have a dynamic deployment based on "classes" and static based on "jar".